src/test/java/org/opendevstack/provision/controller/DefaultControllerTest.java

```diff
@@ -44,35 +45,31 @@
 import org.springframework.web.context.WebApplicationContext;
 
 @SpringBootTest
-@ActiveProfiles({"crowd", "utestcrowd", "quickstarters"})
+@ActiveProfiles({"crowd", "quickstarters"})
 public class DefaultControllerTest {
 
   @MockBean private IJobExecutionAdapter jobExecutionAdapter;
 
   @MockBean private StorageAdapter storageAdapter;
 
-  @MockBean private CrowdAuthenticationManager crowdAuthenticationManager;
+  @Mock private CrowdAuthenticationManager crowdAuthenticationManager;
 
   @Autowired private DefaultController defaultController;
 
-  @Autowired private WebApplicationContext context;
-
   @Autowired private ISCMAdapter realBitbucketAdapter;
 
   @Autowired private IBugtrackerAdapter realJiraAdapter;
 
   @Autowired private ICollaborationAdapter realConfluenceAdapter;
 
+  @Autowired private WebApplicationContext context;
+
   private MockMvc mockMvc;
 
   @BeforeEach
   public void setUp() {
-
-    mockMvc =
-        MockMvcBuilders.webAppContextSetup(context)
-            // .apply(SecurityMockMvcConfigurers.springSecurity())
-            .build();
-
+    // mockMvc without spring security
+    mockMvc = MockMvcBuilders.webAppContextSetup(context).build();
```

Comment:
This test actually needs spring security... I run it with `@AutoConfigureMockMvc` and all test passed. Let us add the annotation `@AutoConfigureMockMvc` instead of line 72.